siftseek/endpoints/seeker/apply/seeker_apply_delete.py

```diff
@@ -0,0 +1,44 @@
+from flask import Response, jsonify, request
+
+from siftseek.endpoints.seeker import seeker
+from siftseek.endpoints.helpers.query_helpers import (
+    get_filtered_job_apps_by_seeker_id_or_404,
+)
+from siftseek.models.db import db
+
+
+@seeker.delete("/apply/<int:seeker_id>")
+def delete_job_applications(seeker_id: int) -> tuple[Response, int]:
```

Comment:
i think you may be copy pasting too quickly. This says you are deleting by job application id but you are passing the `seeker_id` as the parameter.

Reply:
This was something that I was struggling a bit with. When to pass info as a part of the endpoint and when to pass it as a query parameter? This method is pointing at the seeker, then from its test data you can see it passes a query that represents the application(s) to delete (since it can do more than one at once).

```python
job_application_data.py

applications_to_delete = "1,2,3,4"
```

This string is passed during the test to the endpoint as a query.

Of course, as I'm writing this I'm discovering that the endpoint is not working as it should, and it should be throwing an error since the test doesn't set up 4 applications to delete.

But overall, the question I have about the form of the endpoint is when to pass something through the URL as a part of the endpoint vs. as a query?

I know that sometimes websites implement this pattern where the ID of something stays static but the associated username can change so that you can avoid broken links if the username changes in the future.

ex:

`http://www.website.com/user/103981/my_cool_username`

`http://www.website.com/user/103981/my_new_username`

If set up correctly, either of these work, even if the user has changed their username. I guess that was what I was thinking about originally, but perhaps I was just resorting to that since I don't have authentication set up yet? Because not all websites - even most - don't have the user id in the URL. Like, when I go to gmail, I don't see my database ID up in the URL, if that makes sense.

Reply:
Couple things here. 
1. Apologize if this comes off blunt but you should try to be a bit more concise in your code review comments. I don't know which parts / how to address everything in your response so I as a reviewer have to spend time figuring out how/what to interpret. It is very "ramble-y" which i am also at fault of as well. So don't worry too much! Just wanted to bring it up. 
2. Are you asking if you should use a query parameter vs a query string for deleting specific app ids? I think that's what you mean when you say "part of the endpoint" vs as a query. My response to that is it depends. If you're deleting multiple then you have to do it through query strings. But bulk deletes can be kind of dangerous operations so just deleting one id at a time via a query parameter makes sense to me.

Reply:
1. No offense taken, it's good feedback. I could have edited and restated that more succinctly.
2. Ok, noted. It seems I am overthinking trying to minimize calls to the database and should just simplify it at least on the first pass.